### ISSUE TITLE
Coordinape for team

### ISSUE THREAD

**Issue by HBesso31** [Urbanika/DeFi-Adoption-IRL#4]
- [x] Gather wallets of team
- [x] Create coordinape
- [x] Design the way it will work regarding distributing the funds of the grant

**Proof of done:**
![Image 1. Coordinape instructions for team](https://github.com/user-attachments/assets/ba9d8866-c2b3-425e-a049-de5fb57ae79f)

Design the way it will work regarding funds distribution:
1. We use coordinape epoch by epoch as instructed in the guide sent to team (shown in Image 1 above).
2. We get a CSV file with the allocations by team members. Humberto makes a report on fund allocation based on the GIVE tokens members dispersed to each other.
3. Humberto posts the results at our WhatsApp group where members can:
a) confirm the allocation done in Coordinape OR b) ask for a change in the allocation of funds.

If b) then a discussion period of 8 days starts at our [Charmverse](https://app.charmverse.io/urbanika/reward-those-in-the-dao-ops-02322752419338925). In the first 3 days team members that demanded a change need to provide arguments on what is the suggested change and why it should be like that through the Allocation modification proposal template available in [Charmverse](https://app.charmverse.io/urbanika/allocation-modification-proposal-4488829364536424). Other members need to counter argument 

In the following 3 days debates and deliberation happen using the Charmverse Comment section. We use consent, so if people do not provide any counter argument, the proposal will pass and the change will be applied. The Author needs to edit the proposal to include the Debate and Deliberation achievements so it reaches consent.

On the 7th day a vote will happen.

In conclusion, discussion period is distributed in the following way:
I. Allocation modification proposal: 1-3 days after Fund allocation post.
II. Debate and deliberation: 3-6 days.
III. Vote: 7th day.
IV. Votes result and modification: 8th day
V. Fund distribution: 1-3 days after vote by the author of the proposal or by anyone in the team. Proposal in multisig needs to be linked to Charmverse Proposal and requires 3/5 signatures to be approved and executed.

--

Once funds distribution has been confirmed a proposal in the multisig (0x43D950b0209b7568089A62174daD7C3A3fDf412C) needs to happen.
